packages/discovery/src/discovery/runDiscovery.ts

```diff
@@ -70,6 +72,11 @@ export async function dryRunDiscovery(
     config.chain,
   )
 
+  const rawConfigWitComments = await configReader.readRawConfigWithComments(
```

Comment:
This is read here, but not used unless the interactive mode is enabled. We can just move the reading of this config into block where interactive mode is enabled.